0_portfolio_input_check_functions.R

```diff
@@ -1529,7 +1531,7 @@ get_average_emission_data <- function(inc_emission_factors) {
   average_sector_intensity <- data.frame()
 
   if (inc_emission_factors) {
-    average_sector_intensity <- read_rda(paste0(analysis_inputs_path, "average_sector_intensity.rda"))
+    average_sector_intensity <- read_rda(paste0(analysis_inputs_path, "/average_sector_intensity.rda"))
```

Comment:
might as well make this `read_rda(file.path(analysis_inputs_path, "average_sector_intensity.rda"))` while we're changing it anyway